apps/stable_diffusion/shark_sd.spec

```diff
@@ -19,6 +19,9 @@ a = Analysis(
     win_private_assemblies=False,
     cipher=block_cipher,
     noarchive=False,
+    module_collection_mode={
+        'gradio': 'py',  # Collect gradio package as source .py files
```

Comment:
I dont even want to know how you figured out this was required...

Reply:
just wasnt collecting some random source files
tried adding metadata and hidden files, nope, ok just gimme it all then